### PR TITLE
fix: models column size

### PR DESCRIPTION
**Pull request by axel7083**
### What does this PR do?

Increase slightly the size of the age and labels column

### What issues does this PR fix or reference?

Fixes https://github.com/containers/podman-desktop-extension-ai-lab/issues/739

### How to test this PR?

Go to models page and see if labels and age are not cut in big enough screen

### CODE REVIEW

**Review by lstocchi — COMMENTED** [review]
I think i never seen this issue, so LGTM but better to wait for @slemeur approval

**Review by benoitf — APPROVED** [review]

**Pull request merged**
